.github/scripts/wipe-azure.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Exit on error
+set -e
```

Comment:
maybe
```suggestion
set -euo pipefail
```
for more safety

`set -u` errors on undefined variables, `set -o pipefail` propagates the errors out of `|` pipelines instead of silently eating them.